ios/brave-ios/Sources/BraveTalk/BraveTalkJitsiTranscriptProcessor.swift

```diff
@@ -0,0 +1,72 @@
+// Copyright 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import Collections
+import Foundation
+import os.log
+
+public class BraveTalkJitsiTranscriptProcessor {
```

Comment:
Seems like you want an `actor` here based on number of `MainActor` additions

Reply:
It has been an actor but @Brandon-T reverted it to a class when integrating with Leo :P I think he ran into some issues